### ISSUE TITLE
how to manipulate the UI

### ISSUE THREAD

**Issue by jayisidoro** [jessesquires/JSQMessagesViewController#987]
in the viewdid load, i have several lines of code that should manipulate the appearance of the collecitonview that is created by JSQMessagesViewController. However, none of the changes are reflecting in the simulator. 

do i need to change the source code in order to accomplish what i would like? or is there a more simple approach to the problem?

thank you!

**Comment by jayisidoro** [comment]
What I have tried to do: give the collectionview a clear background, along with the navigation bar in order to have an image as the backdrop (which is interchangeable).

Another thing I tried to do was remove the navigation bar, which I was successful in doing, but was not able to change the height of the collectionview in order to create my own custom navigation header.

lastly I've tried changing the height and look of the tool bar, i've gone into the JSQMessages pod and changed lines in an attempt to see results, but have had zero luck.

Any suggestions on how to do any of these goals?

Thank you!
